content/SCALETutorials/Apps/CommunityApps/InstallNextCloudMedia.md

```diff
@@ -7,12 +7,16 @@ tags:
 - media
 ---
 
+
+{{< hint type=important title="Application Tutorials" >}}
+{{< include file="/static/includes/AppsUnversioned.md" >}}
+{{< /hint >}}
+
+The instructions in this article apply to SCALE 22.12.0 and later.
```

Comment:
Does this appear inconsistent with the unversioned messaging above?
```suggestion

```

Reply:
Yes and no. these tutorials are seriously out of date and we've covered this with statements like this one. In this case, the tutorial information dates back to the 22.12.0 release. The solution is to update the instructions to at least DF then we can eliminate these version statements.

Reply:
Nextcloud app tutorial is one we need to get updated to current UI. We can opt to backport the changes to DF but not to Cobia.